api/v1beta1/disruption_webhook.go

```diff
@@ -95,7 +95,8 @@ var _ webhook.Validator = &Disruption{}
 
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (r *Disruption) ValidateCreate() error {
-	logger.Debugw("validating created disruption", "instance", r.Name, "namespace", r.Namespace)
+	logger := logger.With("instance", r.Name, "namespace", r.Namespace)
+	logger.Debugw("validating created disruption", "spec", r.Spec)
```

Comment:
❤️ could/should we also add this to the reconcile loop or will it be "too much logs"?

Reply:
👍